scheduler/src/cook/mesos/share.clj

```diff
@@ -60,14 +60,14 @@
    \"default\" user. If there is NO \"default\" value for a specific type,
    return Double.MAX_VALUE."
   [db user]
-  (->> (util/get-all-resource-types db)
+  (->> [:cpus :mem]
```

Comment:
I'm not sure if you'd like to handle this differently. With the original code, a variety of tests broke, as GPU shares are not handled by the downstream consumers of this code.